app/models/ui/NextPageUrl.scala

```diff
@@ -12,17 +12,8 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *@
+ */
 
-@import models.ui._
+package models.ui
```

Comment:
As this really models the data from a form submission perhaps a different package name would be appropriate. This would separate the submission data from the view components.

Reply:
Im following the pattern here of cest and intelectual property

Reply:
As other applications.